python/publish-packages.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+
```

Comment:
maybe we can rename this file to emphasize that we use this script for Microsoft packages

Reply:
Fixed

Reply:
https://github.com/citusdata/tools/commit/be5591732f65aef151977d4523eb5ba560f5d5fd